go/coinstacks/cosmos/api/api.go

```diff
@@ -45,7 +45,7 @@ type API struct {
 }
 
 func New(httpClient *cosmos.HTTPClient, grpcClient *cosmos.GRPCClient, wsClient *cosmos.WSClient, blockService *cosmos.BlockService, swaggerPath string) *API {
-	r := mux.NewRouter()
+	router := mux.NewRouter()
```

Comment:
I'll rename the one-letter vars as I go along if you don't mind. Concise is good but one letter seems a bit too far

Reply:
this is a golang naming convention. you can see example docs: https://github.com/gorilla/mux#examples

the idea is that variables close to where they are used and where they are just struct references can be labeled very concisely. same reason handler function convention is single letter as well (ex. `func (h *Handler) StartWebsocket() error`)

Reply:
Makes sense 👍